docs/UserGuide.md

```diff
@@ -382,8 +382,14 @@ Format: `edit_archived STUDENT_ID [id/STUDENT_ID] [n/NAME] [e/EMAIL] [g/GRADE] [
 
 * Edits the details of an archived student with the specified `STUDENT_ID`.
 * At least one of the optional fields indicated within the square bracket must be provided.
-* The `...` after the `GROUP_NAME` indicates that multiple `GROUP_NAME` can be provided for editing.
 * The student with the specified `STUDENT_ID` must exists in the archived list.
+* `Student_ID` should **start with A follow by 7 digits and ends with a letter [A-Z]**.
```

Comment:
I think this part will have to be changed after #230 is merged in. For that I think you can just copy the descriptions for student_id, name, email requirements etc later on.

Reply:
ok!